nexus/db-model/src/instance_state.rs

```diff
@@ -33,7 +44,7 @@ impl InstanceState {
     pub fn label(&self) -> &'static str {
         match self {
             InstanceState::Creating => "creating",
-            InstanceState::NoVmm => "no VMM",
+            InstanceState::NoVmm => "no_VMM",
```

Comment:
@gjcolombo N.B. that I swapped the ` ` for a `_` here so that it was easier to parse in the CLI; I wanted to also change the `Display` impl so that they would round-trip. Hope that's okay!

Reply:
Yup, I figured that's what this was for. Works for me!